### PR TITLE
First message visible: hello world!

### PR DESCRIPTION
**Pull request by Alberaan**

### CODE REVIEW

File: alonerpg/templates/home.html [hunk 1]
```diff
@@ -0,0 +1 @@
+{{ salutation }}
```
Comment (jrclaramunt) [inline alonerpg/templates/home.html]: Add new line at the end of the file :)

File: alonerpg/alonerpg/views.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+from django.views.generic.base import TemplateView
+
+class HomePageView(TemplateView):
+
+    template_name = "home.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['salutation'] = "Hello world"
+        return context
```
Comment (jrclaramunt) [inline alonerpg/alonerpg/views.py]: Same here

**Review by jrclaramunt — CHANGES_REQUESTED** [review]

File: alonerpg/templates/home.html (hunk 1 above)
Comment (Alberaan) [reply alonerpg/templates/home.html]: Ok!

File: alonerpg/alonerpg/views.py (hunk 2 above)
Comment (Alberaan) [reply alonerpg/alonerpg/views.py]: ok!

**Review by jrclaramunt — APPROVED** [review]
💯

**Pull request merged**